src/snowflake/snowpark/column.py

```diff
@@ -915,8 +915,17 @@ def __invert__(self) -> "Column":
         return Column(Not(self._expression), _ast=expr, _emit_ast=_emit_ast)
 
     def _cast(
-        self, to: Union[str, DataType], try_: bool = False, _emit_ast: bool = True
+        self,
+        to: Union[str, DataType],
+        try_: bool = False,
+        _emit_ast: bool = True,
+        is_rename: bool = False,
+        is_add: bool = False,
```

Comment:
can we name this closer to sql side names i.e. `rename_fields`, `add_fields`?

Reply:
sure